config.ini

```diff
@@ -49,7 +49,8 @@ sphinx_extensions = sphinx.ext.intersphinx
 header_links = Courses, https://courses.d2l.ai, fas fa-user-graduate,
                All Notebooks, https://github.com/aws-samples/d2l-java, fas fa-download,
                Slack, https://deepjavalibrary.slack.com/#/, fab fa-slack,
-               GitHub, https://github.com/awslabs/djl, fab fa-github,
+               Book's Repo, https://github.com/awslabs/djl, fab fa-github,
+               DJL's Repo, https://github.com/awslabs/djl-java, fab fa-github,
```

Comment:
Please update the links:
```
Book's Repo, https://github.com/aws-samples/d2l-java
DJL's Repo, https://github.com/awslabs/djl
```

Reply:
you can use 
![image](https://user-images.githubusercontent.com/25657787/91204338-82630880-e736-11ea-84cc-310585834825.png)
next time. It is more convient for others.

Reply:
Here:
![image](https://user-images.githubusercontent.com/25657787/91204677-00271400-e737-11ea-8ed5-3f6f537c45e9.png)